internal/builders/go/main.go

```diff
@@ -55,7 +55,7 @@ func runBuild(dry bool, configFile, evalEnvs string) error {
 	if err != nil {
 		return err
 	}
-	fmt.Println(cfg)
+	fmt.Fprintln(pkg.Output, cfg)
 
 	gobuild := pkg.GoBuildNew(goc, cfg)
```

Comment:
Suggestion: Create a `GoBuildConfig` struct or something similar that has a `GoCompilerPath` string and `Output` writer fields and pass it here in lieu of `goc`.

```go
buildConfig := &GoBuildConfig{
   GoCompilerPath: goc,
   Output: os.Stdout,
}
gobuild := pkg.GoBuildNew(buildConfig, cfg)
```

The goal is to encapsulate using the `os.Stdout` writer to `main()` and avoid the use of global package level variables. I think that would make the code cleaner, easier to test, and easier to modify in the future.